portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,25 +24,67 @@
 import java.util.ArrayList;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/data")
-public class DataServlet extends HttpServlet {
 
-    ArrayList<String> mylist = new ArrayList<String>();
-        
+ class Comment{
+    private String name;
+    private String lastname;
+    private String comment;
 
-    public DataServlet(){
-        mylist.add("Hello this");
-        mylist.add("is a");
-        mylist.add("test");
-    }
-    
+    public void setName(String name_){name = name_;}
+    public void setLName(String lastname_){lastname = lastname_;}
+    public void setComment(String comment_){comment = comment_;}
+}
+
+class CheckEmpty{
+    private boolean empty = false;
+
+    public void setEmpty(boolean empty_){empty = empty_;}
+    public boolean isEmpty(){return empty;}
+}
```

Comment:
I'm not quite sure why you need this? Shouldn't the get be fine even if the user doesn't input a comment?

Reply:
Yeah, I think i'm just going to remove CheckEmpty from the code